nxdrive/client/local_client.py

```diff
@@ -613,12 +613,9 @@ def delete(self, ref: str) -> None:
             send2trash(os_path)
         except OSError as exc:
             log.error(f"Cannot trash {os_path!r}")
-            try:
+            with suppress(WindowsError):
```

Comment:
`WindowsError` may not be the good exception to skip. We can have an `ValueError`, `TypeError` or even `AttributeError`. Perhaps filtering on `Exception` is safer?